### DISCUSSION TITLE
"Leaving" shared documents

### DISCUSSION THREAD

**Discussion by threema-danilo** [outline/outline#7882] [Ideas]
When someone shares a document with me, it show up in the "Shared with me" section of the sidebar.

However, once I've given feedback on the shared document, there's no way for me to "leave" that document without having to contact the author of the document.

Is there a way to remove myself from the shared document, or as an alternative to remove it from the "shared with me" section?

**Comment by tommoor** [comment]
You're right, if you don't have permission to view the sharing dialog then currently you're kind of trapped – we'll get that changed.
